api/handlers/handlers.go

```
@@ -0,0 +1,170 @@
+		http.Error(w, "Invalid request method", http.StatusMethodNotAllowed)
+		return
+	}
+
+	positions, err := storage.GetPositions()
+	if err != nil {
+		http.Error(w, "Database error", http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	json.NewEncoder(w).Encode(positions)
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)